httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/AuthenticationHandler.java

```diff
@@ -121,8 +121,7 @@ public boolean isChallenged(
     }
 
     /**
-     * Determines whether the given response represents an authentication challenge, without
-     * changing the {@link AuthExchange} state.
+     * Determines whether the response is 401/407 response depending to the challengeType
```

Comment:
@stoty Can we avoid using explicit status codes here and use a more generic "challenge" term?